rustsec/src/advisory/metadata.rs

```diff
@@ -81,4 +82,12 @@ pub struct Metadata {
     /// This can be used to soft-delete advisories which were filed in error.
     #[serde(default)]
     pub withdrawn: Option<Date>,
+
+    /// License of the advisory content
+    #[serde(default)]
+    pub license: License,
+
+    /// Attribution for the advisory, required for some licenses
+    #[serde(default)]
+    pub attribution: Vec<String>,
```

Comment:
not sure about the representation here, maybe we want to be able to give a URI too

Reply:
cc attribution doc https://wiki.creativecommons.org/wiki/License_Versions#Detailed_attribution_comparison_chart

Reply:
So I think we need to ability to document links.

Reply:
Let's not have aliases - From UX PoV this is very confusing. Long strings are hard to read and prone to errors.